cohttp-lwt-jsoo/src/cohttp_lwt_jsoo.ml

```diff
@@ -180,6 +180,12 @@ struct
 
   let callv ?ctx:_ _uri _reqs = Lwt.fail Cohttp_lwt_xhr_callv_not_implemented
 
+  (* Here we ask for the advice of the maintainers of the library,
+     as we advocate that this function will never be called from
+     this library, but we want to receive a second opinion. *)
+  let call_with_closefn ?ctx:_ ?headers:_ ?body:_ ?chunked:_ _meth _uri =
```

Comment:
Is it just a place holder? I'd rather create a github comment as soon as the PR is open, instead of adding this comment in the code.

Reply:
I was not decided between a comment in the code and a comment in the PR, but I guess this makes more sense. I will also apply this to the `EOF` PR, where we have another "placeholder" instance.

Reply:
Removed the comment and will apply it when I publish the PR.

Reply:
Great, thanks!